FinanceSchedulerDemo/FinanceSchedulerDemo/ClientApp/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I don't think you need vscode specific files in the repository. Please gitignore them

Reply:
It seems that this file is still unignored